posts/virtual_environments.md

```diff
@@ -0,0 +1,91 @@
+# Virtual Environments in Data
+*by [Iaroslav Zeigerman](https://github.com/izeigerman)*
+
+We start this post by setting context for what environments are and how they are used in software today. If you want to skip this part and jump right in please proceed to the [Environments in Data](#environments-in-data) section.
+
+## Environments in Software
+Environments have always been a way for software engineers to test and promote their changes in a safe manner, reducing this way the possibility of causing regressions in production. Generally each environment represents a set of isolated resources to which the software in question is deployed.
+
+Over time the industry settled on a certain set of environments that is now considered to be the best practice:
+- **Testing** - used for testing the bleeding edge of the target software. This environment is updated frequently with most recent code changes. Deployments to this environment are low risk since it's only used by developers of a specific team and is never exposed to end users.
```

Comment:
```suggestion
- **Development** - used for testing the bleeding edge of the target software. This environment is updated frequently with most recent code changes. Deployments to this environment are low risk since it's only used by developers of a specific team and is never exposed to end users.
```